Python unit test failure in test_flexible_type

Reproduced on Linux on Python 3.6. May reproduce elsewhere.

```
_____________________ FlexibleTypeInference.test_int_float _____________________

self = <turicreate.test.test_flexible_type.FlexibleTypeInference testMethod=test_int_float>

    def test_int_float(self):
        verify_inference([IntegerValue], int)
        verify_inference([IntegerValue, IntegerValue], int)
>       verify_inference([IntegerValue, FloatValue], float)

test_flexible_type.py:221: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test_flexible_type.py:198: in verify_inference
    inferred_type, result = _get_inferred_column_type(v_list)
turicreate/cython/cy_flexible_type.pyx:2141: in turicreate.cython.cy_flexible_type._get_inferred_column_type
    ???
turicreate/cython/cy_flexible_type.pyx:2084: in turicreate.cython.cy_flexible_type.flex_list_from_typed_iterable
    ???
turicreate/cython/cy_flexible_type.pyx:1325: in turicreate.cython.cy_flexible_type.fill_typed_list
    ???
turicreate/cython/cy_flexible_type.pyx:1283: in turicreate.cython.cy_flexible_type._fill_typed_sequence
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   OverflowError: Python int too large to convert to C long

turicreate/cython/cy_flexible_type.pyx:1559: OverflowError
```

--- a/src/unity/python/turicreate/cython/cy_flexible_type.pyx
+++ b/src/unity/python/turicreate/cython/cy_flexible_type.pyx
@@ -370,13 +370,13 @@ try:
     import six
     _code_by_type_lookup[<object_ptr>(builtins.dict)]                = FT_DICT_TYPE
     _code_by_type_lookup[<object_ptr>(builtins.float)]               = FT_FLOAT_TYPE
-    _code_by_type_lookup[<object_ptr>(builtins.int)]                 = FT_INT_TYPE
-    _code_by_type_lookup[<object_ptr>(builtins.bool)]                = FT_INT_TYPE  + FT_SAFE
+    _code_by_type_lookup[<object_ptr>(builtins.int)]                 = FT_INT_TYPE + FT_SAFE
+    _code_by_type_lookup[<object_ptr>(builtins.bool)]                = FT_INT_TYPE + FT_SAFE
     _code_by_type_lookup[<object_ptr>(builtins.list)]                = FT_LIST_TYPE
     _code_by_type_lookup[<object_ptr>(builtins.str)]                 = FT_STR_TYPE
     _code_by_type_lookup[<object_ptr>(builtins.tuple)]               = FT_TUPLE_TYPE
     if six.PY2:
-        _code_by_type_lookup[<object_ptr>(builtins.long)]            = FT_INT_TYPE  + FT_SAFE
+        _code_by_type_lookup[<object_ptr>(builtins.long)]            = FT_INT_TYPE + FT_SAFE
         _code_by_type_lookup[<object_ptr>(builtins.unicode)]         = FT_UNICODE_TYPE
 except ImportError:
     pass

